test/chains/nativeVideoChain.test.js

```diff
@@ -0,0 +1,258 @@
+const assert = require('assert');
+const {
+	nativeVideo,
+	nativeVideoAssert,
+} = require('../../lib/chains/nativeVideoChain');
+const {ELEMENT_PROP} = require('../../lib/constants/element');
+const {PROP_COMPARATOR} = require('../../lib/constants/comparator');
+
+describe('Native video chain', () => {
+	it('should have all necessary modifiers', () => {
+		const chain = nativeVideo();
+
+		assert.strictEqual(typeof chain.clone, 'function');
+		assert.strictEqual(typeof chain.abandon, 'function');
+		assert.strictEqual(typeof chain.toString, 'function');
+		assert.strictEqual(typeof chain.then, 'function');
+		assert.strictEqual(typeof chain.not, 'undefined');
+		assert.strictEqual(typeof chain.doesNot, 'undefined');
+		assert.strictEqual(typeof chain.isNot, 'undefined');
+		assert.strictEqual(typeof chain.exist, 'undefined');
+		assert.strictEqual(typeof chain.exists, 'undefined');
+		assert.strictEqual(typeof chain.visible, 'undefined');
+		assert.strictEqual(typeof chain.match, 'function');
+		assert.strictEqual(typeof chain.matches, 'function');
+		assert.strictEqual(typeof chain.matchRepo, 'undefined');
+		assert.strictEqual(typeof chain.matchesRepo, 'undefined');
+		assert.strictEqual(typeof chain.matchJS, 'undefined');
+		assert.strictEqual(typeof chain.matchesJS, 'undefined');
+		assert.strictEqual(typeof chain.isPlaying, 'function');
+		assert.strictEqual(typeof chain.isStopped, 'function');
+		assert.strictEqual(typeof chain.isPaused, 'function');
+		assert.strictEqual(typeof chain.timeout, 'function');
+
+		assert.strictEqual(chain.with, chain);
+		assert.strictEqual(chain.it, chain);
+		assert.strictEqual(chain.should, chain);
+		assert.strictEqual(chain.times, chain);
+	});
+
+	it('should have only allowed modifiers after match is applied', () => {
+		const chain = nativeVideo().match(ELEMENT_PROP.ID, 'someId');
```

Comment:
Can't match by id. This is not an error really, but a dab idea to show-off a wrong property in unit tests - in case someone would look here for examples.